doc/conf.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+
+from pathlib import Path
+from json import loads as json_loads
+from yaml import load as yaml_load, Loader as yaml_Loader
+from tabulate import tabulate
+
+ROOT = Path(__file__).resolve().parent
+
+
+# -- Generate 2019.inc and 2019/*.rst ----------------------------------------------------------------------------------
+
+with open(ROOT.parent / "issues/LCS2019.yml", "r", encoding="utf-8") as stream:
+    LCS2019 = yaml_load(stream, Loader=yaml_Loader)
+
+(ROOT / '2019').mkdir(exist_ok=True)
+
+with open(ROOT / "2019.inc", "w", encoding="utf-8") as index_fptr:
+    index_fptr.write('''\
+.. toctree::
+  :caption: 2019
+  :hidden:
+
+''')
+
+    for key, val in LCS2019.items():
+        index_fptr.write(f'  2019/{key}\n')
+        with open(ROOT / f"2019/{key}.rst", "w", encoding="utf-8") as item_fptr:
+            item_fptr.write(f'''\
+{key}
+{'#'*len(key)}
+
+{val['title']}
+
+{val['body']}
+''')
+
+# -- General configuration ---------------------------------------------------------------------------------------------
+
+extensions = [
+    "sphinx.ext.extlinks",
+    "sphinx.ext.intersphinx",
+]
+
+source_suffix = {
+    ".rst": "restructuredtext",
+}
+
+master_doc = "index"
+project = "VHDL Compliance Tests"
```

Comment:
Somehow, the "documentation logo" is composed of the documentation title, the revision and the word documentation. I can't see this text originating from the `conf.py`. Maybe it's a "feature" of `furo`.

![image](https://user-images.githubusercontent.com/956109/221693837-3d325ea9-1a6e-42f0-8063-c9cd7cd37b71.png)

We can solve this later, but still strange that it adds `latest` and `documentation` to that logo / title.

Reply:
Agree. Thanks something we need to look into. For now, I'm just adding furo with no options, in order to move from the RTD/BTD theme. However, furo supports showing the logo only (as we did before in most projects). See, for instance: https://urllib3.readthedocs.io/en/stable/ or https://www.attrs.org/en/stable/.

Other features I would like to bring:

- The "edit" button at the top of the page.
- The commit and build_id/branch shown in the footer.
- Optionally hiding the heading in the home page.
- Fixing some minor issues when `numfig = True` is used.